From felipe.balbi@nokia.com  Tue Dec 22 11:55:26 2009
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Tue, 15 Dec 2009 23:19:52 +0200
Subject: usb: otg: isp1301_omap: fix compile error
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Tony Lindgren <tony@atomide.com>, linux-usb@vger.kernel.org, Felipe Balbi <felipe.balbi@nokia.com>
Message-ID: <1260911992-27840-1-git-send-email-felipe.balbi@nokia.com>


commit 91c8a5a9985d5bf9c55f6f82f183f57b050b2a3a broke
compilation of this driver after it introduced
otg_init() as a static inline in <linux/usb/otg.h>

Reported-by: Tony Lindgren <tony@atomide.com>
Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/otg/isp1301_omap.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/otg/isp1301_omap.c
+++ b/drivers/usb/otg/isp1301_omap.c
@@ -843,7 +843,7 @@ static irqreturn_t omap_otg_irq(int irq,
 
 static struct platform_device *otg_dev;
 
-static int otg_init(struct isp1301 *isp)
+static int isp1301_otg_init(struct isp1301 *isp)
 {
 	u32 l;
 
@@ -1275,7 +1275,7 @@ static int __exit isp1301_remove(struct 
 static int isp1301_otg_enable(struct isp1301 *isp)
 {
 	power_up(isp);
-	otg_init(isp);
+	isp1301_otg_init(isp);
 
 	/* NOTE:  since we don't change this, this provides
 	 * a few more interrupts than are strictly needed.
